pkg/package.go

```diff
@@ -57,7 +57,7 @@ type Package struct {
 
 	Imports []Import
 	// SourceFiles []File
-	// Qualifier   Qualifier
+	Qualifier   Qualifier
```

Comment:
Is this being actively detected, or do I have to write a function to detect Production vs Development?

Reply:
I don't think these are implemented yet.